plutus-tx-plugin/test/IsData/Spec.hs

```diff
@@ -67,6 +70,23 @@ isDataRoundtrip a =
         unsafeRoundtrip = IsData.unsafeFromBuiltinData d P.== a
     in safeRoundtrip && unsafeRoundtrip
 
+AsData.asData [d|
+  data SecretlyData = FirstC () | SecondC Integer
+  |]
+
+AsData.asData [d|
+  data RecordConstructor a = RecordConstructor { x :: a, y :: Integer }
```

Comment:
There should be a test case testing the field accessors, `x` and `y`.

Reply:
added